presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/RecordingHiveMetastore.java

```diff
@@ -253,6 +256,13 @@ public void updatePartitionStatistics(String databaseName, String tableName, Str
         return loadValue(allTablesCache, databaseName, () -> delegate.getAllTables(databaseName));
     }
 
+    @Override
+    public List<String> getTablesMatchingParameter(String databaseName, String parameterKey, String parameterValue)
+    {
+        ImmutableList<String> keyTuple = ImmutableList.of(databaseName, parameterKey, parameterValue);
```

Comment:
```suggestion
        List<String> keyTuple = ImmutableList.of(databaseName, parameterKey, parameterValue);
```

Reply:
I think you should use `List` here in order to comply with code style in Presto